lib/index.js

```diff
@@ -21,6 +21,7 @@ class PWMetrics {
     this.url = url;
     this.opts = opts;
     this.opts.disableCpuThrottling = this.opts.disableCpuThrottling || false;
+    this.opts.showChart = (typeof this.opts.showChart !== 'undefined') ? (this.opts.showChart == 'true') : true;
```

Comment:
Not sure if this is the best way to do this, however it seems that setting `--showChart=false` in CLI passes the `showChart` option through as a string instead of a bool. I thought about automatically parsing this in `cli.js`, however I think this warrants a different PR altogether if you'd like to go in that direction. This tidbit forces a boolean value no matter what. Would love to hear if you have a better suggestion on how to handle this more gracefully.